.env.example

```diff
@@ -5,9 +5,9 @@
 
 # If you are using docker to manage your postgresql database,
 # you will want to uncomment these
-# PGHOST=localhost
-# PGUSER=postgres
-# PGPASSWORD=password
+PGHOST=localhost
+PGUSER=postgres
+PGPASSWORD=password
```

Comment:
I think we only want to uncomment the first two; as `password` is not guaranteed to be their postgresql password.